package.json

```diff
@@ -55,5 +55,6 @@
     "@semantic-release/git": "^10.0.1",
     "husky": "^8.0.2",
     "semantic-release": "^19.0.5"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Hmm.. not sure how the yarn ended up here. Guess I might have done a yarn install at some point? Does it matter?